src/Build/NuGet.Build.Packaging.Tasks/NuGet.Build.Packaging.targets

```diff
@@ -63,6 +63,11 @@ Copyright (c) .NET Foundation. All rights reserved.
 			<IsPackable>$(IsPackable)</IsPackable>
 			<IsNuGetized>$(IsNuGetized)</IsNuGetized>
 		</TargetPathWithTargetPlatformMoniker>
+		<!-- Design Time is annoying and uses a different output item group to return this. Also adust that -->
+		<_MainProjectOutput>
```

Comment:
It's the item type in the MS.Common targets, so I'd say the VC++ targets need to not-break them by redefining them in an incompatible way.

In the meantime, you can always re-declare the broken target in your own targets and make it compatible again with the MS.Common targets.

It would be good to get the VC++ guys in the loop. Dunno who to @ though